README.md

```diff
@@ -260,4 +260,4 @@ chat.susi.ai/
 
 ## License
 
-This repository is under a GNU LESSER GENERAL PUBLIC LICENSE 2.1.
+This repository is under a GNU LESSER GENERAL PUBLIC LICENSE 2.1, see the [LICENSE.md](https://github.com/fossasia/chat.susi.ai/blob/master/LICENSE) file for details
```

Comment:
@Shehanka it would be better to use only (LICENSE) instead of the longer link

Reply:
That link is won't show up in view mode it's only visible LICENSE.md part only check out markdown cheatsheet [Link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links)
like above link what I mentioned